src/main/java/de/muenchen/dave/domain/Kalendertag.java

```diff
@@ -0,0 +1,29 @@
+package de.muenchen.dave.domain;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+import java.time.LocalDate;
+
+@Entity
+@Data
+@ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
+public class Kalendertag extends BaseEntity {
+
+    @Column(nullable = false, unique = true)
+    private LocalDate datum;
+
+    @Column
+    private String ferientyp;
+
+    @Column
+    private String feiertag;
+
+    @Column(nullable = false)
+    private Integer tagestyp;
```

Comment:
Beim Attribut `tagestyp` den Enum `de.muenchen.dave.domain.enums.TagesTyp` verwenden.

Des Weiteren den Enumwert mit der Annotation `@Enumerated(EnumType.STRING)` als String in der Datenbank hinterlegen.